src/lib/call_mutations.cc

```diff
@@ -57,7 +57,8 @@ CallMutations::CallMutations(const RelationshipGraph &graph, params_t params)
     one_mutation_prior_ = CalculateDNP().prob();
 
     // Calculate prior non-ref allele freq
-    alt_freq_prior_ = HaploidPrior(2)(1);
+    haploid_prior_[1] = HaploidPrior(2);
```

Comment:
Remove this line. as `haploid_prior_[1]` will be set by the constructor in the base class.

Reply:
Makes sense, will do.